srcpkgs/qt-virt-manager/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'qt-virt-manager'
+pkgname=qt-virt-manager
+version=0.71.95
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="qt5-multimedia-devel qt5-svg-devel qtermwidget-devel \
+	libglib-devel spice-gtk-devel libvncserver-devel \
```

Comment:
Indention with just one space and \ at the end of lines inside quotes variables is not required.